src/apps/loyalty/analytics.ts

```diff
@@ -0,0 +1,79 @@
+import { extend, pick } from "lodash"
+import * as sharify from "sharify"
+
+import { ResponseLocalData } from "./types"
+const data = sharify.data as ResponseLocalData
+
+const Analytics = require("analytics-node")
+
+let analytics
+if (data.SEGMENT_WRITE_KEY) {
+  analytics = new Analytics(data.SEGMENT_WRITE_KEY, { flushAt: 1 })
```

Comment:
`flushAt: 1` means instead of batching, we just send the data immediately. I think batching is more appropriate for server-side analytics.